### DISCUSSION TITLE
Reduce spacing under profile photo and description

### DISCUSSION THREAD

**Discussion by PkleinRNA** [alshedivat/al-folio#2600] [Q&A] [answered]
Hi all, 

How can I reduce the space under my profile photo and description underneath? And also reduce the size of the text. Thanks so much.

P

**Comment by george-gca** [comment, marked as answer]
The easiest way to find this and try in advance where you could add the change is by using [chrome dev tools](https://developer.chrome.com/docs/devtools/css) or [firefox dev tools](https://firefox-source-docs.mozilla.org/devtools-user/). In there you can click in the element and find all the attributes that are set for that element and where are they. For more information on how to use this, check [this video](https://www.youtube.com/watch?v=l0sgiwJyEu4). For changing fonts check #2505.

To apply them you should add the changes to `_sass/_base.scss`.
